src/functionalTest/resources/1111006_3.metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+  "po_box": "BULKSCANPO",
+  "jurisdiction": "BULKSCAN",
+  "delivery_date": "2018-06-23T00:00:00.000Z",
+  "opening_date": "2018-06-24T00:00:00.000Z",
+  "zip_file_createddate": "2018-02-24T00:00:00.000Z",
+  "zip_file_name": "$$zip_file_name$$",
+  "envelope_classification": "exception",
```

Comment:
should be different classification is it not?

Reply:
yeah, updated now.